### PR TITLE
fix: article card height on feedback

### PR DESCRIPTION
**Pull request by omBratteng**
### Describe what this PR does
- fixes the card height change when feedback is shown

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1873 #done

### CODE REVIEW

File: packages/shared/src/components/cards/ArticlePostCard.tsx [hunk 1]
```diff
@@ -91,7 +91,10 @@ export const ArticlePostCard = forwardRef(function PostCard(
             onMenuClick={(event) => onMenuClick?.(event, post)}
             onReadArticleClick={onReadArticleClick}
           />
-          <CardTitle lineClamp={showFeedback ? 'line-clamp-2' : undefined}>
+          <CardTitle
+            lineClamp={showFeedback ? 'line-clamp-2' : undefined}
+            className={!showFeedback && 'min-h-[4.875rem]'}
```
Comment (omBratteng) [inline packages/shared/src/components/cards/ArticlePostCard.tsx:96]: I wanted to use `3lh`, but [firefox does not support](https://caniuse.com/mdn-css_types_length_lh) that. So 4.875rem is 3x1.625rem which is the lineheight for CardTitle.

Comment (rebelchris) [reply packages/shared/src/components/cards/ArticlePostCard.tsx:96]: Does the whole card not have a fixed height already?

Comment (omBratteng) [reply packages/shared/src/components/cards/ArticlePostCard.tsx:96]: I see now that card has a min-height of `22.5rem`. Going to see if I can increase that by a bit instead

Comment (rebelchris) [reply packages/shared/src/components/cards/ArticlePostCard.tsx:96]: I think ideally that's fine, maybe the upvote just adds too much somewhere?

Comment (omBratteng) [reply packages/shared/src/components/cards/ArticlePostCard.tsx]: Bumped it up to 23.75rem

**Review by rebelchris — APPROVED** [review]

**Pull request merged**